app/controllers/application_controller.rb

```diff
@@ -1,2 +1,7 @@
 class ApplicationController < ActionController::Base
+  before_action :current_user
+
+  def current_user
+    @current_user ||= User.first
+  end
```

Comment:
- [ ] Please kindly add a LikesController action file for
      - User should be able to add a like to posts.
      - The likes controller should have a `create` action.
      - Update the routes file.
  As per project requirements
  
<img width="939" alt="Screenshot 2023-06-24 at 03 33 54" src="https://github.com/Luchy-B/blogapp/assets/55596970/c874597d-16fa-4623-90f5-83f628d0deb6">